Apps/Common/src/AccessManagement/Authentication/IAuthenticationDelegate.cs

```diff
@@ -1,5 +1,5 @@
 //-------------------------------------------------------------------------
-// Copyright © 2019 Province of British Columbia
```

Comment:
This will introduce smells into your code as our check is currently for 2019.  We should figure out how to make the check 20xx or leave it at 2019.

Reply:
let's leave at 2019 for now. I will edit.